docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md

```diff
@@ -121,155 +129,157 @@ For setting relevant configuration, refer to the
+| Activation-TanH                | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| ArgMax                         | Supported     | Supported\*\* | Supported     | Not Supported | Not Supported | Supported     |
+| Asin                           | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Asinh                          | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Atan                           | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Atanh                          | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| BatchNormalization             | Supported     | Supported     | Supported     | Not Supported | Supported     | Supported     |
+| BinaryConvolution              | Supported     | Supported     | Not Supported | Not Supported | Not Supported | Supported     |
+| Broadcast                      | Supported     | Supported\*\* | Supported     | Not Supported | Supported     | Supported     |
+| Ceil                           | Supported     | Supported\*\* | Supported     | Not Supported | Supported     | Supported     |
+| Concat                         | Supported     |Supported\*\*\*| Supported     | Supported     | Supported     | Supported     |
+| Const                          | Supported     | Supported     | Supported     | Supported     | Supported     | Not Supported |
+| Convolution-Dilated            | Supported     | Supported     | Supported     | Not Supported | ?     | Supported     |
+| Convolution-Dilated 3D         | Supported     | Supported     | Not Supported | Not Supported | ? | Not Supported |
+| Convolution-Grouped            | Supported     | Supported     | Supported     | Not Supported | ?     | Supported     |
+| Convolution-Grouped 3D         | Supported     | Supported     | Not Supported | Not Supported | ? | Not Supported |
+| Convolution-Ordinary           | Supported     | Supported     | Supported     | Supported\*   | ?     | Supported     |
+| Convolution-Ordinary 3D        | Supported     | Supported     | Not Supported | Not Supported | ? | Not Supported |
+| Cos                            | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Cosh                           | Supported     | Supported\*\* | Not Supported | Not Supported |Supported\*\*\*\*| Supported     |
+| Crop                           | Supported     | Supported     | Supported     | Supported     | ?     | Supported     |
+| CTCGreedyDecoder               | Supported\*\* | Supported\*\* | Supported\*   | Not Supported |Supported\*\*\*\*| Supported     |
+| Deconvolution                  | Supported     | Supported     | Supported     | Not Supported | ? | Supported     |
```

Comment:
There are no specs for Deconvolution and Crop.
However I've found mention of a Crop layer in the [issue](https://github.com/openvinotoolkit/openvino/issues/27) created in 2018